Cargo.toml

```diff
@@ -56,6 +56,7 @@ members = [
     "language/vm/vm-runtime/vm-cache-map",
     "language/vm/vm-runtime/vm-runtime-types",
     "language/vm/vm-genesis",
+    "libra-dev",
```

Comment:
should we make libra-dev its own workspace ? That way we can explicitly control all the dependencies.

@bmwill  for comment

Reply:
Making it its own workspace has no impact on controlling the deps, its still trivially easy to provide a path dep. So people do not move it into another workspace.